documentation/man/features/build.rst

```diff
@@ -16,6 +16,8 @@ SYNOPSIS
 | *kw* (*b* | *build*) [\--llvm] [\--alert=(s | v | (sv | vs) | n)]
 | *kw* (*b* | *build*) [(-c | \--clean)] [\--alert=(s | v | (sv | vs) | n)]
 | *kw* (*b* | *build*) [(-f | \--full-cleanup)] [\--alert=(s | v | (sv | vs) | n)]
+| *kw* (*b* | *build*) [(-v | \--verbose)] [\--alert=(s | v | (sv | vs) | n)]
```

Comment:
Add `--verbose` in the `bash_autocomplete` and `_kw`.

Reply:
It seems like `--alert` is a legacy option, which no longer works now. So no need to add it here.